### PR TITLE
Keycloak how to guide on adding a new user

### PR DESCRIPTION
**Pull request by bnallapeta**

### CODE REVIEW

File: content/how-to-guides/keycloak.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
+# Setting Up User Access in Keycloak for MTO Console
+
+This guide walks you through the process of adding new users in Keycloak and granting them access to Multi Tenant Operator (MTO) Console.
+
+## Accessing Keycloak Console
+
+a. Log in to the OpenShift Console.
```
Comment (SheryarButt) [inline content/how-to-guides/keycloak.md:7]: Would prefer bullets instead of a,b,c.

File: content/how-to-guides/keycloak.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+## Accessing Keycloak Console
+
+a. Log in to the OpenShift Console.  
+b. Go to the 'Routes' section within the 'multi-tenant-operator' namespace.
+
+![openshift-routes](../images/openshift-routes.png)
+
+c. Click on the Keycloak console link provided in the Routes.
+d. Login using the admin credentials (default: admin/admin).
```
Comment (SheryarButt) [inline content/how-to-guides/keycloak.md:13]: c and d are rendering on the same line for some reason.

File: content/how-to-guides/keycloak.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+
+## Adding new Users in Keycloak
+
+a. In the Keycloak console, switch to the `mto` realm.  
+b. Go to the `Users` section in the `mto` realm.  
+c. Follow the prompts to add a new user.
+
+![keycloak-new-user](../images/keycloak-new-user.png)
```
Comment (SheryarButt) [inline content/how-to-guides/keycloak.md:21]: A highlight on the image for where to select realm would better.

File: content/how-to-guides/keycloak.md [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+
+![keycloak-users](../images/keycloak-users.png)
+
+## Accessing MTO Console
+
+a. Go back to the OpenShift Console, navigate to the Routes section, and get the URL for the MTO Console.
```
Comment (SheryarButt) [inline content/how-to-guides/keycloak.md:27]: **a** and **b** rendering on the same line.

**Review by SheryarButt — CHANGES_REQUESTED** [review]

File: content/how-to-guides/keycloak.md (hunk 1 above)
Comment (bnallapeta) [reply content/how-to-guides/keycloak.md:7]: Bullets was causing issues. That's why switched to a,b,c.

Let me try again.

File: content/how-to-guides/keycloak.md (hunk 2 above)
Comment (bnallapeta) [reply content/how-to-guides/keycloak.md:13]: Fixed it already. it was in the second commit.

File: content/how-to-guides/keycloak.md (hunk 4 above)
Comment (bnallapeta) [reply content/how-to-guides/keycloak.md:27]: fixed it in the second commit.
